internal/registry/registry.go

```diff
@@ -91,6 +92,19 @@ func (r *registry) VerifyModuleExists(layer v1.Layer, pathPrefix, kernelVersion,
 	return err == nil
 }
 
+func (r *registry) GetDigest(ctx context.Context, image string, tlsOptions *kmmv1beta1.TLSOptions, registryAuthGetter auth.RegistryAuthGetter) (string, error) {
+	pullConfig, err := r.getPullOptions(ctx, image, tlsOptions, registryAuthGetter)
+	if err != nil {
+		return "", fmt.Errorf("failed to get pull options for image %s: %w", image, err)
```

Comment:
Do we really need %w here?
```suggestion
		return "", fmt.Errorf("failed to get pull options for image %s: %v", image, err)
```